db/migrate/20220810155757_create_posts.rb

```diff
@@ -12,3 +12,9 @@ def change
     end
   end
 end
+
+
+
+fourth_user = User.create(name: 'clinton', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'teacher')
+
+first_post = Post.create(author: fourth_user, title: 'Hello', text: 'This is my first post')
```

Comment:
- [ ] When I tried to run the migration command I got this error please it would be better if you fix it 
Hint: that happened because the associations weren't set up
<img width="1136" alt="Screen Shot 2022-08-11 at 11 07 36" src="https://user-images.githubusercontent.com/61790587/184118953-e9f285b5-9902-4618-9d14-a35e31b20928.png">